File: PawnshopAPI/Entities/TransactionItems.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PawnshopAPI.Entities
{
    public class TransactionItems
    {
        public int itemId { get; set; }
        public int previousTransactionId { get; set; }
        public int trackingId { get; set; }
        public string category { get; set; }
        public string categoryDescription { get; set; }
        public string itemDescription { get; set; }
        public int appraisalValue { get; set; }
        public int sellingPrice { get; set; }
        public bool isSold { get; set; }
        public string dateSold { get; set; }
        public string newDateTransaction { get; set; }
    }
}

File: PawnshopWEB/src/app/_model/transaction/transaction-items.ts
import { ItemAuditTrail } from "../item/item-audit-trail";

export interface TransactionItems{
    itemId:number;
    previousTransactionId:number;
    trackingId:number;
    category:string;
    categoryDescription:string;
    // itemDescription:string
    description:string
    appraisalValue:number;
    sellingPrice:number;
    isSold:boolean;
    dateSold?:string;
    newDateTransaction:string; //insert new date of new transaction to update the date expire
    itemAuditTrails:ItemAuditTrail[];
}
